css/sm_screen.css

```diff
@@ -0,0 +1,112 @@
+#menu {
+	display: none;
+}
+
+.header_logo {
+  padding-left: 30px;
+}
+
+.header_list {
+	padding-right: 30px
+}
+
+.menuIcon {
+	display: flex;
+	font-size: 28px;
+	color: #6abfc3;
+}
+
+.menuIcon i {
+	color: #6abfc3;
+}
+
+#sectionOne {
+	display: grid;
+	grid-template-rows: 1fr 1fr;
+}
+
+.bottomSectionTwo #articleWrapperSectionTwo {
+	display: grid;
+	grid-template-rows: 1fr 1fr 1fr;
+	grid-template-columns: 1fr;
+}
+
+.sectionTwoImg img {
+	height: 131px;
+}
+
+.topSectionTwo {
+	padding: 0;
+	margin: 0;
+}
+
+#sectionTwo h4 {
+	margin: 0;
+}
+
+#articleWrapperSectionThree {
+	display: grid;
+	grid-template-rows: 1fr 1fr 1fr 1fr;
+	grid-template-columns: 1fr 1fr;
+	grid-column-gap: 20px;
+	grid-row-gap: 20px;
+}
+
+#sectionThreeLastBox {
+	display: grid;
+	grid-column: 1 / 3;
+	justify-content: center;
+	align-items: center;
+	width: 100%;
+	background-color: #f3f3f3;
+}
+
+#sectionThreeLastBox .lastBoxWrapper {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	width: 250px;
+	height: 240px;
+	background-color: white;
+}
+
+.sectionThreeBottom  {
+	margin: 0;
+	padding: 0;
+	height: 100px;
+}
+
+.sectionFourBoxWrapper {
+	display: grid;
+	grid-template-rows: 1fr 1fr 1fr 1fr;
+	grid-template-columns: 1fr;
+	grid-column-gap: 20px;
+	grid-row-gap: 40px;
+	width: 85%;
+	grid-column: 1 / 3;
+}
+
+/* Footer */
+
+#footerLeft {
+	display: none;
+}
+
+footer {
+	display: grid;
+	grid-template-rows: 1fr;
+	grid-template-columns: 1fr 1fr;
+}
+
+#footerCenter li {
+	margin-left: 10px;
+  margin-right: 10px;
```

Comment:
#### `REQUIRED`

Please, fix the indentation. :blush:
A good indentation is fundamental for good readability of the code.
Many IDE  can help you with this task.
This is VS Code, for example as shown below using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode):

![Screenshot of how to format code using VS Code](https://res.cloudinary.com/bolaah/image/upload/v1563045186/github-microverse-project/code-reviews/code-review002.gif)

### DO this for all occurrences.